### ISSUE TITLE
gam print cigroup-members throws  KeyError: 'memberKey'

### ISSUE THREAD

**Issue by jay-eleven** [taers232c/GAMADV-XTD3#198]
Hi Ross!

I found an error while printing cigroup members:

```
jay@cloudshell:~$ gam print cigroup-members
Getting all Cloud Identity Groups that match query (All Cloud Identity Groups), may take some time on a large Google Workspace Account...
Got 3 Cloud Identity Groups: test@test.com - abuse@test.com
Getting all Members, Managers, Owners for groups/aaa (1/3)
Getting all Members, Managers, Owners for groups/bbb (2/3)
Traceback (most recent call last):
  File "__init__.py", line 57923, in ProcessGAMCommand
  File "__init__.py", line 27812, in doPrintCIGroupMembers
  File "__init__.py", line 25930, in mapCIGroupMemberFieldNames
KeyError: 'memberKey'
```

I've tried to debug it but I reached nowhere...  😅

**Comment by taers232c** [comment]
Fixed 6.08.08

Thanls for all the Wiki cleanup.

Ross
--
***@***.***

**Comment by jay-eleven** [comment]
Works like a charm! Thanks Ross!

PS: Thanks for being **so** responsive!
